docs/OV_Runtime_UG/migration_ov_2_0/common_inference_pipeline.md

```diff
@@ -0,0 +1,195 @@
+# Inference Pipeline {#openvino_2_0_inference_pipeline}
+
+Usually to inference model with the OpenVINO™ Runtime an user needs to do the following steps in the application pipeline:
+- 1. Create Core object
+- 2. (Optional) Read model from the disk
+ - 2.1. (Optional) Model preprocessing
+- 3. Load the model to the device
+- 4. Create an inference request
+- 5. Fill input tensors with data
+- 6. Start inference
+- 7. Process the inference results
+
+Code snippets below cover these steps and show how application code should be changed for migration to OpenVINO™ Runtime 2.0.
+
+## 1. Create Core
+
+Inference Engine API:
+
+@snippet docs/snippets/ie_common.cpp ie:create_core
+
+OpenVINO™ Runtime API 2.0:
+
+@snippet docs/snippets/ov_common.cpp ov_api_2_0:create_core
+
+## 2. (Optional) Read model from the disk
```

Comment:
Not sure that we should state that this step is 'optional' for such high-level documentation. Technically it is correct, but user must be quite experienced and if so, such user already knows that it is 'optional' step if you already have ov::Model from somewhere. Suggesting to remove (Optional)

Reply:
done